### DISCUSSION TITLE
Quarto Book: Put footnote on chapter header

### DISCUSSION THREAD

**Discussion by kruschke** [quarto-dev/quarto-cli#7841] [Q&A] [answered]
### Description

(This is copied from [my post on Stack Overflow](https://stackoverflow.com/questions/77614329/quarto-book-put-footnote-on-chapter-header).)

I want to put a footnote on chapter headers in Quarto book format. It seems to work fine when rendered to PDF, but does crazy things when rendered to HTML.

Specifically, suppose these are chapter headers in two Quarto book files:

```
# A Chapter Header^[Footnote text for A Chapter Header.] {#sec-a_chapter}
```

```
# B Chapter Header[^ch_head] {#sec-b_chapter}


[^ch_head]: Footnote text for B Chapter Header.
```

They work fine when rendered to PDF. But when rendered in HTML, multiple copies of the footnotes appear, and the footnotes appear on the `index.qmd` page, and the header footnotes get incorporated into the chapter titles in the table of contents.

**How can footnotes be put on chapter headers, and render properly in both PDF and HTML?**

----------

*A complete, minimal example is below. Because it is a Quarto book project, it has multiple files.*

File `_quarto.yml`:
```
project:
  type: book

book:
  title: "Quarto Book Test"
  author: "Firstname Lastname"
  chapters:
    - index.qmd
    - a_chapter.qmd
    - b_chapter.qmd

format:
  html:
    theme: default
  pdf:
    documentclass: book
```

File `index.qmd`:
```
# Preface {.unnumbered}

This is a Quarto book.

**There are no footnotes specified on this page.**
```

File `a_chapter.qmd`:
```
# A Chapter Header^[Footnote text for A Chapter Header.] {#sec-a_chapter}

This chapter header has a footnote with `^[text]` format.

This sentence has a footnote^[Text of sentence footnote.]

Check chapter links: See @sec-b_chapter.
```

File `b_chapter.qmd`:
```
# B Chapter Header[^ch_head] {#sec-b_chapter}

[^ch_head]: Footnote text for B Chapter Header.

This chapter header has a footnote with `[^label]`, `[^label]: text` format.

This sentence has a footnote^[Text of sentence footnote.]

Check chapter links: See @sec-a_chapter.
```

----------

**Here are screenshots of output in HTML.**

The table of contents with **un**desired footnotes in the chapter headers:

[![enter image description here][1]][1]

The preface with mysterious footnotes:

[![enter image description here][2]][2]

Chapter 1 with mysterious footnotes:

[![enter image description here][3]][3]

----------

  [1]: https://i.stack.imgur.com/PZA89.png
  [2]: https://i.stack.imgur.com/uxKje.png
  [3]: https://i.stack.imgur.com/nnvQ2.png

**Comment by luifrancgom** [comment, marked as answer]
I add #8023 this issue so the developers could fix it.

**Reply by cderv** [reply]
Thank you

**Comment by kruschke** [comment]
Thanks for posting the bug report.
